docs/setup-and-installing.md

```diff
@@ -23,7 +23,7 @@
 ### Setting up your machine
 
 1. Install prerequisites:
-	- [Unity 2018.1.1](https://unity3d.com/get-unity/download/archive)
+	- [Unity 2018.2.0b10](https://unity3d.com/unity/beta-download)
```

Comment:
Good job finding [/beta-download](https://unity3d.com/unity/beta-download), that's a much nicer experience than [get-unity/download/archive](https://unity3d.com/get-unity/download/archive).